frontend/src/features/properties/map/__snapshots__/MapView.test.tsx.snap

```diff
@@ -1,920 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`MapProperties View Renders the map 1`] = `
```

Comment:
looking into why the snapshots for these were lost.

Reply:
Most likely due to the rename: "MapProperties View Renders the map" -> "MapView Renders the map"